.github/workflows/qc-prebuild.yml

```diff
@@ -10,3 +10,11 @@ jobs:
       uses: actions/checkout@v3
     - name: Run RustFMT Check
       run: cd src && cargo fmt --check
+  rust-clippy:
+    name: Rust-Clippy Check
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Checkout source
+      uses: actions/checkout@v3
+    - name: Run Rust-Clippy Check
+      run: cd src && cargo clippy
```

Comment:
Better to merge it with a rustfmt job as a step so we won't have too many jobs to show in the PR.

Reply:
Ah, no multi-thread job? But then doesn't that make 
`Pre-Build / Rustfmt Check`
Be untrue since we use multiple commands?

Reply:
The parallel is not necessary for this case because it not taking much time for each step.